sklearn/preprocessing/data.py

```diff
@@ -270,8 +293,16 @@ def transform(self, X):
         check_is_fitted(self, 'scale_')
 
         X = check_array(X, copy=self.copy, ensure_2d=False)
-        X *= self.scale_
-        X += self.min_
+
+        if len(X.shape) == 2:
```

Comment:
I think the original contributors of the code intended this code to work on 1-d arrays: https://github.com/scikit-learn/scikit-learn/pull/5002/files#diff-5ebddebc20987b6125fffc893f5abc4cR268

There are also test cases to make sure that 1-d arrays can be scaled.